Gemfile

```diff
@@ -0,0 +1,3 @@
+source "https://rubygems.org"
+
+gem "fastlane"
```

Comment:
Can we pin this to a semver minor?

Reply:
Lockfile does it?

Reply:
See this -> https://stackoverflow.com/questions/9265213/should-i-specify-exact-versions-in-my-gemfile

It may be better to use the pessimistic locking operator, because whilst the lockfile does provide a default source of installation, updates to dependencies will not end up being constrained